pom.xml

```diff
@@ -80,11 +84,11 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>3.2.5</version>
-           <configuration>
+           <!-- <configuration>
             <suiteXmlFiles>
                   <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
             </suiteXmlFiles>
-      </configuration>
+      </configuration> -->
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/selenium-automation/pull/2#issuecomment-2155458326>#5765cd</a><span> - 06/07/2024, 07:54 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The configuration block for the maven-surefire-plugin is commented out. This disables the execution of tests specified in the testng.xml during the Maven test phase, which can lead to untested code being deployed.
</div>
<div id="fix">
<b>Fix:</b> Uncomment the configuration block to ensure that tests are executed as part of the build process.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div>
<pre><code>
 @@ -87,5 +87,5 @@
 -           &lt;!-- &lt;configuration&gt;
 +           &lt;configuration&gt;
              &lt;suiteXmlFiles&gt;
                    &lt;suiteXmlFile&gt;src/test/resources/testng.xml&lt;/suiteXmlFile&gt;
              &lt;/suiteXmlFiles&gt;
 -      &lt;/configuration&gt; -->
 +      &lt;/configuration&gt;
</code></pre>
</div>
</details>
</div>
<br>


</div>